train.py

```diff
@@ -25,14 +23,20 @@
 num_features = context_embeddings.shape[2]
 raw_question = questions[0][0]['question']
 
-words_in_context = tf.placeholder(tf.float32, [time_steps, batch_size, num_features])
-lstm = tf.keras.layers.LSTM(units=3)
-lstm_forward = lstm(words_in_context)
 
-question_embedding_placeholder = tf.placeholder(tf.float32, [batch_size, 512])
+question_embedding_use = embed_model([raw_question])
+dense_linear = tf.keras.layers.Dense(units=num_features)
+question_embedding = dense_linear(question_embedding_use)
+question_embedding = tf.reshape(question_embedding, [batch_size, 1, num_features])
+
+context_embedding = tf.placeholder(tf.float32, [time_steps, batch_size, num_features])
+
+embedded_question_context = tf.concat([question_embedding, context_embedding], axis=0)
```

Comment:
the dimensions of the concatenation operation are not consistent. On axis 0, `question_embedding` has dimension `batch_size` while `context_embedding` has dimension `time_steps`.

Reply:
I have in the latest push changed the dimensions to be consistent with the ordering `[batch_size, time_steps, num_features]`